go.mod

```diff
@@ -11,7 +11,8 @@ require (
 	github.com/onosproject/onos-api/go v0.7.66
 	github.com/onosproject/onos-lib-go v0.7.12
 	github.com/onosproject/onos-test v0.6.5
-	github.com/smartystreets/assertions v1.0.0 // indirect
+	github.com/prometheus/common v0.4.1
```

Comment:
I don't know why this dep should show up here? Perhaps run go mod tidy.

Reply:
go mode tidy solved it :ok_hand: